introduction.md

```diff
@@ -0,0 +1 @@
+on the wat to becoming a github master 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.